### PR TITLE
add prettier rules to eslint

### PR DESCRIPTION
**Pull request by ArjixWasTaken**

### CODE REVIEW

**Comment by ArjixWasTaken** [comment]
how are there already conflicts?

**Comment by ArjixWasTaken** [comment]
fucking renovate bot...

**Comment by ArjixWasTaken** [comment]
I ain't resolving those conflicts, renovate is annoying for PRs.........

**Comment by ArjixWasTaken** [comment]
does it make sense to merge renovate PRs so frequently?
it's not as if they are included in a release as frequently, so they don't benefit the users

File: package.json [hunk 1]
```diff
@@ -229,12 +229,15 @@
     "electron-vite": "2.3.0",
     "esbuild": "0.23.1",
     "eslint": "9.10.0",
+    "eslint-config-prettier": "^9.1.0",
```
Comment (JellyBrick) [inline package.json:232]:
```suggestion
    "eslint-config-prettier": "9.1.0",
```

version pinning

**Review by JellyBrick — CHANGES_REQUESTED** [review]

**Pull request closed without merging**